src/components/ClearAllTags.tsx

```diff
@@ -4,10 +4,11 @@ const ClearAllTags = (props: {
   classNames: {
     clearAll: string,
   },
+  accessibilityLabel?: string,
   onClick: () => void,
 }) => {
   return (
-    <button className={props.classNames.clearAll} onClick={props.onClick}>
+    <button aria-label={props.accessibilityLabel} className={props.classNames.clearAll} onClick={props.onClick}>
```

Comment:
let's rename the prop to simply `aria-label` here

Reply:
keeping it in camelCase and renamed to `ariaLabel ` https://github.com/react-tags/react-tags/blob/76cb9fffab90225487cb4bd2c614f0161e6d450a/src/components/ClearAllTags.tsx#L7